tools/ansible/tasks/wsu/test/e2e/wsu_test.go

```diff
@@ -135,4 +139,17 @@ func testFilesCopied(t *testing.T) {
 		assert.Emptyf(t, stdout.String(), "Missing file: %s", fullPath)
 	}
 
+	// Check the SHA of kube.tar.gz downloaded
+	kubeTarPath := ansibleTempDir + "\\" + "kube.tar.gz"
+	command := fmt.Sprintf("certutil -hashfile %s SHA512", kubeTarPath)
```

Comment:
How are you guaranteeing that `certutil` is available?

Reply:
The documentation isn't particularly great in this regards, but looks like certutil is part of default os installation from Windows 7 onwards.
[A forum answer](https://social.technet.microsoft.com/Forums/en-US/09fb7a70-69c2-4cc0-ad82-173c06774261/availbility-of-certutil-on-different-windows-versions?forum=winserversecurity) is the closest I could find.

Reply:
Then please add a comment about it.